services/utils.py

```
@@ -94,22 +153,47 @@ def load_json(self, filename: str, date: Optional[datetime] = None) -> Optional[
+        
+        Args:
+            file_path: Path to the persisted file to delete
+            
+        Returns:
+            True if file was deleted successfully, False if file didn't exist
+        """
+        try:
+            if file_path.exists():
+                file_path.unlink()
+                return True
+            return False
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)